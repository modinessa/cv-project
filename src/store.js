import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
