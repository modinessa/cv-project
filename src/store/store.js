import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});