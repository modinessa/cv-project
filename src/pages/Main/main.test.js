import { Provider } from 'react-redux';
import {BrowserRouter, Route, Routes} from 'react-router-dom'; 
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Main } from './index';

describe('Main', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<BrowserRouter>
					<Routes>
						<Route path='/test' element={<Main/>}/>
					</Routes>
				</BrowserRouter>
			</Provider>
		)).toMatchSnapshot();
	})
})