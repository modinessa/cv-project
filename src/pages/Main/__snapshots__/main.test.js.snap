// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders component 1`] = `null`;
